x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/CancellationTests.java

```diff
@@ -231,4 +231,8 @@ private ClusterService mockClusterService(String nodeId) {
         when(mockClusterService.getClusterName()).thenReturn(mockClusterName);
         return mockClusterService;
     }
+
+    private static IndexResolver indexResolver(Client client) {
```

Comment:
Since it's another test where this code is used, how about moving it in EqlTestUtils.

Reply:
I think there's a similar call in SQL's cancelation tests only (hope I didn't miss other place), but not sure how this code could be shared between them.